js/index.js

```diff
@@ -0,0 +1,28 @@
+const iroh = require('../index');
+
+(async () => {
+  console.log('hello', iroh);
+
+  const node = await iroh.IrohNode.withPath("./iroh-node");
+  console.log("hello iroh", node.nodeId());
+
+  const blobs = await node.blobsList();
+  console.log("any blobs?", blobs);
+
+  const author = await node.authorCreate();
+  console.log("author created", author.toString());
+
+  await node.blobsAddFromPath(
+    "/Users/dignifiedquire/opensource/iroh-ffi/js/index.js",
```

Comment:
if you use `__filename` instead here, the code will run successfully on other computers

Reply:
but it works on my machine 😛